lstchain/io/io.py

```diff
@@ -396,7 +395,7 @@ def write_array_info(event, output_filename):
                         )
                         continue
 
-            camera.to_table().write(
+            camera.geometry.to_table().write(
                 output_filename,
                 path=f'/instrument/telescope/camera/{camera_name}',
```

Comment:
at the moment, what is the `camera_name` we are writing to the df: `LSTCam` or `LSTCam-004`? I think that regardless of the version, we are writing `LSTCam`, right? If so, we should consider changing that

Reply:
We were writing LSTCam regardless of the version, exact.
Let me check the status with this PR.

Reply:
Still is `LSTCam`

Reply:
I do not think we should write the same name, specially with the big difference that we have between `LSTCam` and `LSTCam-00x`, but not a showstopper here